.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Describe the Problem and Solution
```

Comment:
Before this line:
<!-- Following this template is optional if you don't feel like it applies to your change feel free to delete it. -->

Reply:
Makes sense. Updated in [c4090d3](https://github.com/wiredtiger/wiredtiger/pull/10973/commits/c4090d3f4983e4a87181dcbdc3254c4c03403a57)

Reply:
We can remove this when we move to the comment solution.

Reply:
Actually with the comment solution we should have something like:
Hi! Thanks for creating a pull request! The below question and checklist are intended to help with verifying your change is well test. Answering the question and filling out the checklist is optional.

Reply:
Updated as suggested